examples/operator/convolution_operator.py

```diff
@@ -0,0 +1,55 @@
+"""Create a convolution operator by wrapping a library."""
+
+import odl
+import scipy
+import scipy.signal
+
+
+class Convolution(odl.Operator):
+    """Operator calculating the convolution of a kernel with a function.
+
+    The operator inherits from ``odl.Operator`` to be able to be used with ODL.
+    """
+
+    def __init__(self, kernel):
+        """Initialize a convolution operator with a known kernel."""
+
+        # Store the kernel
+        self.kernel = kernel
+
+        # Initialize the Operator class by calling its __init__ method.
+        # This sets properties such as domain and range and allows the other
+        # operator convenience functions to work.
+        super(Convolution, self).__init__(
+            domain=kernel.space, range=kernel.space, linear=True)
+
+    def _call(self, x):
+        """Implement calling the operator by calling scipy."""
+        return scipy.signal.fftconvolve(self.kernel, x, mode='same')
+
+    @property
+    def adjoint(self):
+        """Implement ``self.adjoint``."""
+        return self  # the adjoint is the same as this operator
+
+
+# Define the space the problem should be solved on.
```

Comment:
Simpler sentence (easier to digest): "Define the space on which the problem should be solved."